Statistical_Inference/Variance/lesson

```diff
@@ -263,7 +263,7 @@
   Hint: Remember the many many examples we went through. The sqrt(n) figured prominently in them.
 
 - Class: text
-  Output: Congrats! You've concluded this vary long lesson on variance. We hope you liked it vary much.
```

Comment:
You're not a bother at all! Sorry if I seem curt in these messages, I just have many tasks I need to get through.